iothub/service/src/RegistryManager.cs

```diff
@@ -406,46 +406,6 @@ public virtual Task<BulkRegistryOperationResult> AddDeviceWithTwinAsync(Device d
             }
         }
 
-        /// <summary>
-        /// Register a list of new devices with the system
-        /// </summary>
-        /// <param name="devices">The Device objects being registered.</param>
-        /// <returns>Returns a string array of error messages.</returns>
-        [Obsolete("Use AddDevices2Async")]
-        public virtual Task<string[]> AddDevicesAsync(IEnumerable<Device> devices)
```

Comment:
**Question**: in this class, we have multiple methods(like `AddDevicesAsync()` ) which have replacements (`AddDevices2Async()`). With the deprecated ones removed, do we wanna rename the replacements to the original ones (`AddDevices2Async()` -> `AddDevicesAsync()`)?
@abhipsaMisra @timtay-microsoft

Reply:
That sounds like a good suggestion.